### DISCUSSION TITLE
Create a Filter or Tag cloud for a field with multi-select enabled?

### DISCUSSION THREAD

**Discussion by kasperbolarsen** [microsoft-search/pnp-modern-search#3083] [Q&A]
As I recall it, creating a filter on a site column (people) with multi-select enabled has always been an issue, but perhaps somebody have found a good workaround?

I have a site column "Subject area expert" of type "Person or group" and it is as per customer requirement set to accept multiple persons.

I have mapped the ows_subjectAreaExpert property to RefinableString102 , added that RefinableString to the results web part, and set the filter in the Refiner web part.

As expected the values shows as a semicolon separated list of names.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/0b12fc83-7408-47d6-a302-f896dda645e1)

The customer is requesting the filter to show one person per line, and when clicked to show those results where that person is one of the people in the subject area expert field.
As far as I know, this is just not possible?

**Comment by till-llit** [comment]
Is this still PNP Search 3.x in your screenshot?

I have pretty much the same requirement running here without a problem - in 4.x there is a checkbox "Multivalues" in the Filter Webpart to enable this use case.

My RefinableString has ; seperated values, my filter is splitting them and allow me to select one person at a time.

**Reply by kasperbolarsen** [reply]
Hi, no, I am on 4.9 :-) I think the Multivalues in the Filter web part refers to the option to select more than one  :
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/e344e178-cccc-4a65-b72a-4c22c0ab1a6d)

However the people are still joined in the items :-(

**Reply by kasperbolarsen** [reply]
@till-llit are you using a custom filter?

**Reply by till-llit** [reply]
not at all, i was searching for my "magic" config, but did not found anything special.

i have a RefineableString for all possible Author Fields (like Author, MailFrom, Mail:6, Office:4 and so on).

when i check the value i can see multiple people separated with semicolon like in your filter.

but my filter is displaying them as single values and i can filter for each unique person.
